Demo/EarlGreyExample/Podfile

```diff
@@ -39,7 +39,7 @@ target TEST_TARGET_SWIFT do
 end
 
 post_install do |installer|
-  load('configure_earlgrey_pods.rb')
```

Comment:
We have a demo project that used the earlgrey gem, if I'm right. We can show them the gem usage there. We don't want this to look like we've supplanted the file altogether.